### PR TITLE
V4.0.3 - translation to simplified chinese( now done

### PR DESCRIPTION
**Pull request by hi-unc1e**
This Pull Request relates to issue #1152 

https://github.com/OWASP/ASVS/issues/1152

### CODE REVIEW

**Comment by elarlang** [comment]
Thank you for PR,

it may take some time now to validate the structure and have some tests with document generator scripts. At least at the moment I expect that it's not smooth way with the charset, bet lets see.

File: 4.0/zh-cn/0x10-V1-Architecture.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+在本章中，ASVS涵盖了任何良好安全架构的主要方面：可用性、保密性、完整性、不可抵赖性和隐私。 这些安全原则中的每一条，都必须适用并内置于所有应用程序中。 “左移”至关重要，从安全编码Checklists、指导和培训、编码和测试、构建、部署、配置和操作开始，到后续的独立测试，确保所有安全控制存在且功能正常。 这最后一步，曾经是我们作为一个行业所做的一切，但当开发人员每天数十次或数百次地推送代码时，就已经不够了。 应用安全专业人员必须跟上敏捷技术的步伐，这意味着要适应开发人员的工具，学习编码，并与开发人员一起工作，而不是在其他人离开的几个月后再批评项目。
+
+## V1.1 安全软件开发生命周期
+
+| # | 说明 | L1 | L2 | L3 | CWE |
+| :---: | :--- | :---: | :---:| :---: | :---: |
+| **1.1.1** | 验证使用安全的软件开发生命周期，在开发的各个阶段解决安全问题。 ([C1](https://owasp.org/www-project-proactive-controls/#div-numbering)) | | ✓ | ✓ | |
+| **1.1.2** | 验证在每次设计变更或sprint计划中使用威胁建模，以识别威胁、计划对策、促进适当的风险响应，并指导安全测试。 | | ✓ | ✓ | 1053 |
+| **1.1.3** | 验证所有用户信息和功能是否包含功能安全约束，例如 “作为一个用户，我应该能够查看和编辑我的个人资料。我不应该能够查看或编辑其他人的资料” | | ✓ | ✓ 1110 |
+ **1.1.4** | 验证应用程序所有的信任边界、组件和重要数据流的文档，判断其合理性。| | ✓ | ✓ | 1059 |
```
Comment (elarlang) [inline 4.0/zh-cn/0x10-V1-Architecture.md:20]: missing | in the beginning

File: 4.0/zh-cn/0x10-V1-Architecture.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+在本章中，ASVS涵盖了任何良好安全架构的主要方面：可用性、保密性、完整性、不可抵赖性和隐私。 这些安全原则中的每一条，都必须适用并内置于所有应用程序中。 “左移”至关重要，从安全编码Checklists、指导和培训、编码和测试、构建、部署、配置和操作开始，到后续的独立测试，确保所有安全控制存在且功能正常。 这最后一步，曾经是我们作为一个行业所做的一切，但当开发人员每天数十次或数百次地推送代码时，就已经不够了。 应用安全专业人员必须跟上敏捷技术的步伐，这意味着要适应开发人员的工具，学习编码，并与开发人员一起工作，而不是在其他人离开的几个月后再批评项目。
+
+## V1.1 安全软件开发生命周期
+
+| # | 说明 | L1 | L2 | L3 | CWE |
+| :---: | :--- | :---: | :---:| :---: | :---: |
+| **1.1.1** | 验证使用安全的软件开发生命周期，在开发的各个阶段解决安全问题。 ([C1](https://owasp.org/www-project-proactive-controls/#div-numbering)) | | ✓ | ✓ | |
+| **1.1.2** | 验证在每次设计变更或sprint计划中使用威胁建模，以识别威胁、计划对策、促进适当的风险响应，并指导安全测试。 | | ✓ | ✓ | 1053 |
+| **1.1.3** | 验证所有用户信息和功能是否包含功能安全约束，例如 “作为一个用户，我应该能够查看和编辑我的个人资料。我不应该能够查看或编辑其他人的资料” | | ✓ | ✓ 1110 |
```
Comment (elarlang) [inline 4.0/zh-cn/0x10-V1-Architecture.md:19]: missing | before CWE column

File: 4.0/zh-cn/0x10-V1-Architecture.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+ **1.1.4** | 验证应用程序所有的信任边界、组件和重要数据流的文档，判断其合理性。| | ✓ | ✓ | 1059 |
+| **1.1.5** | 验证应用程序的高级架构及远程连接服务涉及的定义和安全分析。 ([C1](https://owasp.org/www-project-proactive-controls/#div-numbering)) | | ✓ | ✓ | 1059 |
+| **1.1.6**| 验证集中、简单（设计）、安全、经过审查、和可重复使用的安全控制措施的实施情况，以避免重复、缺失、无效或不安全的控制措施。（[C10](https://owasp.org/www-project-proactive-controls/#div-编号）| | ✓ | ✓ | 637 |
```
Comment (elarlang) [inline 4.0/zh-cn/0x10-V1-Architecture.md:22]: incorrect link, C10

**Comment by elarlang** [comment]
Started to validate the document structure and I have some feedback to give.

1. Please keep the same style and syntax through the document.

Example (different translation and format for the same thing):

`[DELETED, DUPLICATE OF 4.1.3]`
`[已删除，与 4.1.3 重复]`

`[DELETED, DUPLICATE OF 12.4.1]`
`[删除，与12.4.1重复]`


2. Please check, that requirement text ends with dot `。` (except some `”` situations)

3. Please keep links like they are, including brackets around them, like:
```
([C1](https://owasp.org/www-project-proactive-controls/#div-numbering))
```

4. In text, sometimes there is separate whitespace character (space) after dot (`。`) and sometimes not. Which one is correct?

5. Check visually, that everything is correct. For example:
<https://github.com/hi-unc1e/ASVS/blob/v4.0.3/4.0/zh-cn/0x10-V1-Architecture.md>

You can detect that:

  * 1.1.3 before CWE columns there is `|` missing and table is incorrect
  * 1.1.6 there is incorrect syntax for markdown and there is broken/incorrect link

**Comment by elarlang** [comment]
ping @hi-unc1e 

Do you need / want some help with correcting markdown structure or you plan to do this part yourself?

File: 4.0/zh-cn/0x10-V1-Architecture.md (hunk 2 above)
Comment (hi-unc1e) [reply 4.0/zh-cn/0x10-V1-Architecture.md]: solved in https://github.com/hi-unc1e/ASVS/commit/50d33c03314c3b81f9ad911d64948e815f1c4b8b#diff-ea4ed61673cf88738bd9b2e414d5ac46f6cbc5da75c40676e4dd63dcb113086a

File: 4.0/zh-cn/0x10-V1-Architecture.md (hunk 1 above)
Comment (hi-unc1e) [reply 4.0/zh-cn/0x10-V1-Architecture.md]: solved in https://github.com/hi-unc1e/ASVS/commit/040ec0749205a4c3e1b4280a22e61fcfe48dc1bd#diff-ea4ed61673cf88738bd9b2e414d5ac46f6cbc5da75c40676e4dd63dcb113086a

File: 4.0/zh-cn/0x10-V1-Architecture.md (hunk 3 above)
Comment (hi-unc1e) [reply 4.0/zh-cn/0x10-V1-Architecture.md]: solved in https://github.com/hi-unc1e/ASVS/commit/bc2853e120cd43416252632d097dbff389c54577

**Comment by tghosth** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|master|[](https://github.com/OWASP/ASVS/pull/1178)|


### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Comment by hi-unc1e** [comment]
> Do you need / want some help with correcting markdown structure or you plan to do this part yourself?

Hi, I've been involving Spring Festival in my hometown, so that it will take some time for the correction.
But, i will make the correction myself, so that I could finish this item in real sense.

Thanks for your feedback.

**Comment by elarlang** [comment]
Actually I made "some" changes already, you can watch the diff from here and please check that everything is correct from language point of view:
https://github.com/OWASP/ASVS/pull/1169/files/b9b6c655d49390bba4b08258c5a34119f4c07b86..f038e33c04b0a46bda7929d4a0c9c4ad2ad69646

Translation/PR is merged to the repo, ported to master, documents generated and everything is done from out side at the moment.

**Pull request merged**